src/routes/safe/components/Transactions/TxList/TxQueueRow.tsx

```diff
@@ -15,15 +22,20 @@ type TxQueueRowProps = {
 export const TxQueueRow = ({ isGrouped = false, transaction }: TxQueueRowProps): ReactElement => {
   const { activeHover } = useContext(TxHoverContext)
   const [tx, setTx] = useState<Transaction>(transaction)
+  const willBeReplaced = tx.txStatus === LocalTransactionStatus.WILL_BE_REPLACED ? ' will-be-replaced' : ''
+  const isPending = useSelector((state: AppReduxState) => isTxPending(state, transaction.id))
+  const pendingTx = useSelector(pendingTxByChain)
+  const pendingTxNonce = isMultisigExecutionInfo(pendingTx?.executionInfo) ? pendingTx?.executionInfo.nonce : undefined
+  const nonce = isMultisigExecutionInfo(transaction.executionInfo) ? transaction.executionInfo.nonce : undefined
 
   useEffect(() => {
-    if (activeHover && activeHover !== transaction.id) {
+    if ((activeHover && activeHover !== transaction.id) || (!isPending && nonce === pendingTxNonce)) {
```

Comment:
If they are both `undefined` this condition will pass. You should check that they both exist as well.